Added data-aid attr to new elements

coreyogburn opened the pull request.

coreyogburn committed:
Added data-aid attr to new elements

jertel commented on html/index.html:
```diff
@@ -370,12 +370,16 @@ <h2 v-text="i18n[category]"></h2>
                   </span>
                   <div v-if="isCategory('detections')" class="manual-sync">
-                    <div>
+                    <div data-aid="detections_manualsync_engine">
```
Can you add a _select suffix on this

coreyogburn replied:
Done

coreyogburn committed:
Tweaks

data-aid name change.

The spans were messing the styles, updated so the buttons still look good.
